hw/top_earlgrey/data/ip/chip_spi_host_testplan.hjson

```diff
@@ -16,9 +16,81 @@
             - Run with min and max SPI clk frequencies and with single, dual and quad SPI modes.
 
             Verify all SPI host instances in the chip.
+
+            Notes for silicon targets:
+            - The testbench that mimics a physical SPI device should measure the clock to verify that the frequencies for each speed mode is correct.
+            This can be done by checking timing of the beginning and the end of the packet.
             '''
+      features: [
+        "SPIHOST.RATE.STANDARD",
+        "SPIHOST.RATE.DUAL",
+        "SPIHOST.RATE.QUAD",
+      ]
       stage: V2
+      si_stage: SV2
       tests: ["chip_sw_spi_host_tx_rx"]
     }
+    {
+      name: chip_sw_spi_host_pass_through
+      desc: '''Verify that the SPI host can be configured in pass through mode.
+
+            - Connect the SPI host to a serial NOR flash device.
+            - Configure an SPI device and that SPI host in pass through mode.
+            - Use the testbench's SPI host to write a word to the flash through the passthrough SPI device and host in OpenTitan.
```

Comment:
I found this difficult to follow! :-) I'm convinced that it's right, but maybe it could be phrased more easily. Would it make sense to explicitly talk about "OpenTitan acting as an SPI device and host"? That might make the three-item "(real) host -> OT -> flash device" chain easier to see.

Reply:
Hopefully this has now been clarified.